src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: stretch;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:not(:last-child) {
+    margin: 0 20px 0 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    color: $nav-text-color;
```

Comment:
Ensure that the variable `$nav-text-color` is defined in your SCSS files. If it's not defined, this will cause a compilation error.